<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenbushi</class>
 <widget class="QMainWindow" name="fenbushi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创能机器人配置软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="fensbushi_num">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_fenbushi">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLabel" name="fenbushi_labal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入现场分布式传感器个数：</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>711</width>
      <height>401</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>709</width>
       <height>399</height>
      </rect>
     </property>
     <widget class="QLabel" name="fensbushi_loc">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>分布式传感器位置</string>
      </property>
     </widget>
     <widget class="QLabel" name="fenbushi_loc2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>分布式传感器对应点位置</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="ok_fenbushinum">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionRobot_info"/>
     <addaction name="actionFenbushi"/>
     <addaction name="actionPlanwork"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRobot_info">
   <property name="text">
    <string>Robot_info</string>
   </property>
  </action>
  <action name="actionFenbushi">
   <property name="text">
    <string>Fenbushi</string>
   </property>
  </action>
  <action name="actionPlanwork">
   <property name="text">
    <string>Planwork</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
